source/mobile/mobile-overview.rst

```diff
@@ -1,37 +1,84 @@
-Mobile Applications Guide
-=========================
+Deploy Mattermost Mobile Apps
+=============================
 
-There are three main options for deploying the Mattermost Mobile Apps.
+This documentation provides foundational information you need when developing a plan for an enterprise mobile application deployment. It’s intended for business leaders, system administrators, and mobile app developers responsible for deploying Mattermost in their organization.
 
-Install from Apple App Store and Google Play
----------------------------------------------
+This documentation provides information to help you: 
 
-For quick and easy deployment, you can download the Mattermost mobile apps directly from the Apple App Store and Google Play. Please see our guide on :doc:`installing from the Apple App Store and Google Play Store <mobile-appstore-install>` for more details. 
+- Determine the ideal mobile deployment model for your organization.
+- Understand what’s required to build your own Mattermost Mobile Apps.
+- Deploy Mattermost Mobile Apps.
 
-This option is recommended for:
+.. note::
 
-1. Testing out the mobile applications
-2. Team Edition servers using no push notifications, or push notifications from Mattermost's :ref:`TPNS <tpns>` (Test Push Notification Service)
-3. Enterprise Edition servers using push notifications from Mattermost's :doc:`HPNS <mobile-hpns>` (Hosted Push Notification Service)
+  - Mattermost Cloud customers must use Mattermost pre-built public apps for mobile deployment. 
```

Comment:
@esethna @enahum - I don't have a firm rationale for this limitation. 

Cloud System Admins don't have access to the Environment section of the System Console (to configure TPNS, HPNS, or a manual push notification service). However, after a quick chat with @marianunez, there doesn't appear to be any reasons why Cloud customers can't build their own mobile apps. 

@enahum - If you agree, I can remove this line from the docs altogether.

Reply:
Not having access to that configuration section probably is a limitation though. Cloud admins are restricted admins and do not have the same level of access as self-managed admins.

Reply:
@marianunez makes an excellent point -- if a Cloud customer wanted to build their own apps, they'd need to host their own push notification service, and specify the server URL in an area of the System Console that Cloud System Admins don't have access to.

Reply:
Discussed via DM. From @enahum: "Technically they can, but apparently cause of the lack of functionality on the system message in cloud they won’t be able to setup the correct push proxy, so short answer seems to be: Cloud = pre-built only".

Reply:
Lol @cwarnermm I did edit my post "System message" -> "system console"